app/views/play/level/tome/SpellView.coffee

```diff
@@ -988,6 +993,37 @@ module.exports = class SpellView extends CocoView
     @spell.transpile()  # TODO: is there any way we can avoid doing this if it hasn't changed? Causes a slight hang.
     @updateAether false, false
 
+  onWebDevError: (error) ->
```

Comment:
This may be the right approach instead of making the logic go through the `displayAether` flow, but we will probably be finding edge cases that would have been solved if we had shared that. So whenever we think of something not working exactly right, it will probably pay to understand when errors would have been set and cleared through `displayAether` and make sure we are at least acknowledging the same cases.